tests/helpers/testStdDefs.cpp

```diff
@@ -0,0 +1,58 @@
+/**
+ *  This program is free software: you can redistribute it and/or modify
+ *  it under the terms of the GNU Lesser General Public License as
+ *  published by the Free Software Foundation, either version 3 of the
+ *  License, or  (at your option) any later version.
+ *
+ *  This program is distributed in the hope that it will be useful,
+ *  but WITHOUT ANY WARRANTY; without even the implied warranty of
+ *  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ *  GNU General Public License for more details.
+ *
+ *  You should have received a copy of the GNU General Public License
+ *  along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ *
+ **/
+
+/**
+ * @file testStdDefs.cpp
+ * @ingroup Tests
+ * @author David Coeurjolly (\c david.coeurjolly@liris.cnrs.fr )
+ * Laboratoire d'InfoRmatique en Image et Systemes d'information - LIRIS (CNRS, UMR 5205), CNRS, France
+ *
+ * @date 2016/01/19
+ *
+ * Functions for testing class StdDefs.
+ *
+ * This file is part of the DGtal library.
+ */
+
+///////////////////////////////////////////////////////////////////////////////
+#include <iostream>
+#include "DGtal/base/Common.h"
+#include "ConfigTest.h"
+#include "DGtalCatch.h"
+#include "DGtal/helpers/StdDefs.h"
+///////////////////////////////////////////////////////////////////////////////
+
+using namespace std;
+using namespace DGtal;
+
+///////////////////////////////////////////////////////////////////////////////
+// Functions for testing class StdDefs.
+///////////////////////////////////////////////////////////////////////////////
+
+TEST_CASE( "Testing StdDefs" )
+{
+  
+  //Just for compilation timings.
+  SECTION("Testing domain instanciation (Z2i)")
+    {
+      Z2i::Domain dom(Z2i::Point(0,0), Z2i::Point(16,16));    
+      REQUIRE( dom.isValid() );
+    }
```

Comment:
I'll redo a timing test but it was not terrific on ```tests/``` maybe because some of them are quite old and do not use Z2i::Z3i...
(i will remove this test)